dashboard/app/controllers/registrations_controller.rb

```diff
@@ -370,4 +370,9 @@ def get_users_to_destroy(user)
     users << {id: user.id, name: user.name}
     users
   end
+
+  def destroy_dependent_users(user)
+    user_ids_to_destroy = get_users_to_destroy(user).pluck(:id)
```

Comment:
Out of curiosity, do you know why `pluck` works here?  Since `get_users_to_destroy` returns an Array rather than an ActiveRecord Collection I assumed you'd have to `.map(&:id)`.  (Question phrased another way in [this two-year-old StackOverflow question](https://stackoverflow.com/questions/21901142/how-to-pluck-email-from-array-of-users).)

Reply:
interesting! i originally had `.map(&:id)`, but found this [ActiveRecord doc](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck) that suggested pluck instead. it looks like the person in that StackOverflow question isn't using ActiveRecord, so that could be why their solution didn't work

Reply:
Oh weird - that doc also only shows `pluck` being called on ActiveRecord Collections, but quickly checking against arrays in irb it seems to work fine.  Turns out [`Enumerable#pluck`](https://www.rubydoc.info/github/rails/rails/Enumerable:pluck) was [added in Rails v5.0.0-beta1](https://github.com/rails/rails/commit/d909211736b1e85cb011a1a3a87c08d1d1f32a63#diff-33d6b41213775779b35c3f31915e9f98).  TIL.